contracts/0.8.25/vaults/LazyOracle.sol

```diff
@@ -229,7 +229,7 @@ contract LazyOracle is ILazyOracle, AccessControlEnumerableUpgradeable {
                 connection.infraFeeBP,
                 connection.liquidityFeeBP,
                 connection.reservationFeeBP,
-                connection.pendingDisconnect
+                vaultHub.isPendingDisconnect(vaultAddress)
```

Comment:
prob need to update oracle to explicitly opt out from calculating fees on pending disconnect conditions